benchmark/buffers/buffer-compare-instance-method.js

```diff
@@ -4,11 +4,11 @@ const common = require('../common.js');
 const bench = common.createBenchmark(main, {
   size: [16, 512, 1024, 4096, 16386],
   args: [1, 2, 3, 4, 5],
-  millions: [1]
+  n: [1]
 });
 
-function main({ millions, size, args }) {
-  const iter = millions * 1e6;
+function main({ n, size, args }) {
+  const iter = n * 1e6;
```

Comment:
Hm, it seems like the description in #18778 was not clear enough: it was not about just replacing the variable name. Instead of having `millions = 1` it should be `n = 1e6`, not `n = 1`. That applies to all of these changes.

This also has influence on the for loops and the benchmark end call. These should be changed accordingly. 

If there is a `iter` variable as here, it is likely obsolete and it could be replaced with `n` instead.

Reply:
working on it.